tests/data/html/issue779.html

```diff
@@ -0,0 +1,248 @@
+    </tr>
+    <tr>
+        <td><strong>Code</strong></td>
+        <td><strong>Student</strong></td>
+        <td class="center" colspan="26"><strong>Grades</strong></td>
+    </tr>
+    <tr>
+    </tr>
+    <tr>
+        <td>0000</td>
+        <td>STUDENT NAME</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">Good</td>
+        <td class="center">Completed</td>
+    </tr>
+    <tr>
+        <td>0000</td>
+        <td>STUDENT NAME</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">8.00</td>
+        <td class="center">Good</td>
+        <td class="center">Completed</td>
+    </tr>
+    <tr lenght="20">
+        <td colspan="31"></td>
+    </tr>
+    <tr>
+        <td class="center" colspan="11">
+            <strong>Location/Date</strong>
+        </td>
+        <td class="center" colspan="10">
+            <strong>Signature</strong>
+        </td>
+        <td class="center" colspan="10">
+            <strong>Course secretary signature</strong>
+        </td>
+    </tr>
+    <tr lenght="70">
+        <td class="center" colspan="11" style="padding: 30px">
+        </td>
+        <td class="center" colspan="10" rowspan="3">
```

Comment:
The issue is eventually caused by non-wellformed HTML, specifically by these two rowspans which have nowhere further to overflow.

Also: empty tr element with td, tr has no lenght attribute (nor length to that matter).

Reply:
You're right. I removed the rowspans and the output worked as expected.I had not realized this detail before.